src/components/common/codeInput/CodeInput.test.tsx

```diff
@@ -144,7 +144,7 @@ describe("CodeInput Component", () => {
     expect(verifyCqlCode).toHaveBeenCalledTimes(1);
   });
 
-  it("Should allow user to enter custom code system and code", async () => {
+  it.skip("Should allow user to enter custom code system and code", async () => {
```

Comment:
unsure if this still works. cant test locally

Reply:
It works: 
![image](https://github.com/user-attachments/assets/68ca08f6-3956-45c4-8213-f6be1481ccf7)